packages/pages/src/components/map/map.tsx

```diff
@@ -1,17 +1,26 @@
-import React, { createContext, useContext, useEffect, useRef, useState } from "react";
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
 import classnames from "classnames";
-import { GoogleMaps, Map as MapType, MapOptions } from "@yext/components-tsx-maps";
+import {
+  GoogleMaps,
+  Map as MapType,
+  MapOptions,
+} from "@yext/components-tsx-maps";
 import { Coordinate, GeoBounds } from "@yext/components-tsx-geo";
-import { MapProps, MapContextType } from "./types";
-import css from "./map.module.css";
+import type { MapProps, MapContextType } from "./types";
```

Comment:
The linter should have complained that this doesn't end in .js. Did it not do that on commit? Do you get errors on `pnpm lint`?

Reply:
No it's not picking up on that file, but I'll add them now.

Reply:
pnpm lint did callout the marker.tsx file through the commit flow and running it manually just called out the index file, which I also missed. Neither though found any issue with the map.tsx file.